package.json

```diff
@@ -36,94 +36,95 @@
   },
   "dependencies": {
```

Comment:
I'd like to see `yaml-js` go back to `0.2.0`. It's currently `^0.2.0`, and 0.2.0 is the [latest version of yaml-js](https://www.npmjs.com/package/yaml-js), so it shouldn't make any difference today, since the same version will be installed either way.

My reasoning for this, is that in semantic versioning, major version 0 (`0.*.*`) can have breaking changes at any time. I don't feel comfortable opening ourselves up to future incompatibility by using `^` as the selector for `yaml-js`'s version.

See my recent PR for this, for more context: https://github.com/swagger-api/swagger-ui/pull/3281